openssl-dynamic/pom.xml

```diff
@@ -161,16 +161,6 @@
       <artifactId>netty-tcnative-classes</artifactId>
       <version>${project.version}</version>
     </dependency>
-    <dependency>
-      <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter-api</artifactId>
-      <version>${junit.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter-engine</artifactId>
-      <version>${junit.version}</version>
-    </dependency>
```

Comment:
hmm.. shouldn't these be still included here but have the right scope ?

Reply:
I think the correct solution would be to have these just declared in a dependency management in the parent and with the right scope here.

Reply:
Note that there are a couple other dependencies declared in the parent as well (netty-build-common and netty-jni-util).  Do you want to leave them in dependencies in the parent, or move them to dependencyManagement in the parent and dependencies in each submodule?

Reply:
lets just merge this one for now and I will think about the rest